src/control/drpc/drpc_client.go

```diff
@@ -121,7 +112,7 @@ func (c *ClientConnection) sendCall(msg *Call) error {
 
 func (c *ClientConnection) recvResponse() (*Response, error) {
 	respBytes := make([]byte, MAXMSGSIZE)
-	numBytes, _, _, _, err := c.conn.ReadMsgUnix(respBytes, nil)
+	numBytes, err := c.conn.Read(respBytes)
```

Comment:
So this is not semantically equivalent. Read will not handle the messaging properly like ReadMsg does. They are two very different calls. I'm not sure if regular conn implements this.

Reply:
Just wanted to note the results of our discussion/research here - read is doing the right thing for sockets under the covers, so the default Read should be OK here. I did test communications manually as well and things do seem to work in both directions.